.buildkite/plutus-script-dump.yml

```diff
@@ -0,0 +1,11 @@
+# A nightly job that runs a cardano node and dumps Plutus script evaluation events
```

Comment:
How do we know it's scheduled nightly? Is that configured in the builkite UI or something?

Reply:
Yep, that's configured in the UI. I believe you can only define steps in the yml file.